sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Utils.java

```diff
@@ -85,11 +100,11 @@ static String getEtag(String cosmosDbPayload) {
      * @param e Exception to display in the log.
      */
     public static synchronized void logApiCallFailure(Exception e) {
-        AppCenterLog.error(Constants.LOG_TAG, "Failed to call App Center APIs", e);
+        AppCenterLog.error(LOG_TAG, "Failed to call App Center APIs", e);
         if (!HttpUtils.isRecoverableError(e)) {
             if (e instanceof HttpException) {
                 HttpException httpException = (HttpException) e;
-                AppCenterLog.error(Constants.LOG_TAG, "Exception", httpException);
+                AppCenterLog.error(LOG_TAG, "Exception", httpException);
```

Comment:
I failed to see the reason why we only log unrecoverable `HttpException` in this util method. I don't see why we are filtering out recoverable `HTTPException`s.

Addressing this is out of scope of this PR, just want to bring it up.

Reply:
Its duplicate log, http client already does this and I noticed I got logs twice in sasquatch. It's because we don't test e2e enough to notice.